packages/api/src/external/cda/generate-ccd.ts

```diff
@@ -47,8 +45,9 @@ export async function generateCcd(patient: Patient, requestId: string): Promise<
     entry: [...metriportGenerated, { resource: fhirOrganization }],
   };
   const parsedBundle = bundleSchema.parse(bundle);
-  const validatedBundle = validateFhirEntries(parsedBundle);
+  uploadCcdFhirDataToS3(patient, parsedBundle, requestId);
```

Comment:
Aren't we missing an `await` here?

Reply:
I figured we don't need to await the upload on the route where the CX just uploads data contributions. We can handle the file upload async

Reply:
Sounds good, by then let's make the error handling on that function encompass the whole logic, otherwise we won't know if something breaks before the try/catch. An alternative would be to add a `.then()` here, but the other one seems better.

Also, this is a good example where a comment is helpful, something like we do in other instances: `// intentionally async`

Reply:
Oh, crap, I actually thought you were referring to a different place in the code. I think here, I will make this await it. And leave everything as is. 

Good point about the `// intentionally async`. 
I'll add those here - [link](https://github.com/metriport/metriport/blob/8e272ef73d2d013cfc0e31ec99a9f951ffb71a8d/packages/api/src/command/medical/patient/handle-data-contributions.ts#L65-L66) and here - [link](https://github.com/metriport/metriport/blob/8e272ef73d2d013cfc0e31ec99a9f951ffb71a8d/packages/api/src/command/medical/patient/handle-data-contributions.ts#L74-L75)